Dockerfile

```diff
@@ -21,6 +22,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /
+COPY templates/ templates/
```

Comment:
Does this suffice ? I saw permission denied

Reply:
@VedantMahabaleshwarkar can you confirm? 

I believe this is it, it worked for us...where did you see permission denied?

Reply:
I didn't see permission denied anywhere. Before the change the Custom Resource had the error  `Missing templates/*, no such file or directory` and adding this line fixed the error.